Dockerfile

```diff
@@ -1,21 +1,9 @@
-FROM hydroshare/hs_docker_base:4.2.x
+FROM hydroshare/hs_docker_base:django-1.11
 MAINTAINER Phuong Doan pdoan@cuahsi.org
 
-### Begin - HydroShare Development Image Additions ###
-RUN pip install --upgrade pip && pip install djangorestframework==3.6.4
-RUN pip install \
-  robot_detection \
-  django-ipware \
-  django-test-without-migrations \
-  django-rest-swagger \
-  jsonschema \
-  nameparser \
-  probablepeople \
-  geopy \
-  hs_restclient==1.2.12 \
-  pyyaml
-
-### End - HydroShare Development Image Additions ###
+# inplaceedit in pip doesn't seem compatible with Django 1.11 yet...
+RUN pip uninstall -y django-inplaceedit
+RUN pip install git+https://github.com/theromis/django-inplaceedit.git@e6fa12355defedf769a5f06edc8fc079a6e982ec
```

Comment:
@sblack-usu Could you update the base image to remove django-inplaceedit so it does not have to be uninstalled here as an extra step? Also my impression is that we are not using django-inplaceedit in HydroShare, so this might be a good chance to test it out and if we are not using it, we don't have to handle this upgrade incompatibility here.

Reply:
I'm building hs_dockerbase without django-inplaceedit, locally now.  If we don't end up needing it, I'll remove both of these lines.

Reply:
we need inplaceedit, on your suggestion, I removed it from hs_dockerbase so we don't have to uninstall it here before installing a different version.

Reply:
looks like another package has django-inplaceedit as a dependency so it still needs to be reinstalled.